src/pybind/chain/chain_training_pybind.cc

```diff
@@ -0,0 +1,51 @@
+// pybind/chain/chain_training_pybind.cc
+
+// Copyright 2019   Microsoft Corporation (author: Xingyu Na)
+
+// See ../../../COPYING for clarification regarding multiple authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+// See the Apache 2 License for the specific language governing permissions and
+// limitations under the License.
+
+#include "chain/chain_pybind.h"
```

Comment:
Guys, is there a good reason why we are using underscores rather than dashes in these filenames?  Doesn't seem very consistent.  The dashes were for google style guide reasons I think.

Reply:
I guess Dongji used this style first, for example, `kaldi_pybind.cc` instead of `kaldi-pybind.cc`.

I followed `kaldi_pybind.cc`.

Reply:
Yep, I followed `kaldi_pybind.cc` too. I think we can switch at certain point later.